### ISSUE TITLE
disallow use of both instantiationPhysical and instantiationDigital

### ISSUE THREAD

**Issue by kvanmalssen** [WGBH/PBCore2.0#89]
Currently the schema allows for both instantiationPhysical and instantiationDigital to be used for the same instantiation. If the elements and their usage stays as is (depending on the outcome of #67), the schema should be modified so that only one of the two is allowed, using xsd:choice. For example:

```
<xsd:choice>
    <xsd:element name="instantiationPhysical" type="sourceVersionStringType"/>
    <xsd:element name="instantiationDigital" type="sourceVersionStringType"/>
</xsd:choice>
```

**Comment by jolene2323** [comment]
Agreed! This has never made sense to me. We use instantiationPhysical for our physical tape instantiations and instantiationDigital for audiofiles on servers.

**Comment by asteino** [comment]
I could see where it would be useful to allow both and disallow both. In addition to disallowing the use of both instantiationPhysical and instantiationDigital, a reference element to another instantiation (either physical or digital) could be added (if there isn't  one already).
